Eduplatform_site/account/models.py

```diff
@@ -0,0 +1,70 @@
+
+import datetime
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+from .mixins import DateTimeMixinModel
+from .managers import CustomUserManager
+
+
+class User(DateTimeMixinModel,AbstractUser):
+    username = models.CharField(_("username"),max_length=150)
+    first_name = models.CharField(_("first name"), max_length=150, blank=True)
+    last_name = models.CharField(_("last name"), max_length=150, blank=True)
+    email = models.EmailField(_("email address"), unique=True)
+    is_staff = models.BooleanField(_("staff status"),default=False,)
+    is_active = models.BooleanField(_("active"),default=True)
+    date_joined = models.DateTimeField(_("date joined"), default=timezone.now)
+
+    objects = CustomUserManager()
+
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = ["password"]
```

Comment:
"password" doesn't need to be here, you may add "username", "first_name", "last_name" fields here